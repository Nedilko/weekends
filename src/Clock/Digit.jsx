import PropTypes from 'prop-types'

Digit.propTypes = {
  digit: PropTypes.string.isRequired,
}

function Digit({ digit }) {
  return <div>{digit}</div>
}

export default Digit
