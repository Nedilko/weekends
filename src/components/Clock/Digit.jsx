import PropTypes from 'prop-types'

Digit.propTypes = {
  digit: PropTypes.number.isRequired,
}

function Digit({ digit }) {
  return <div className="text-5xl">{digit}</div>
}

export default Digit
