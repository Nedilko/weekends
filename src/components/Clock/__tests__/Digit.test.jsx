import Digit from '@components/Clock/Digit'
import { render, screen } from '@utils/test-utils'

describe('Digit', () => {
  it('should render digit', () => {
    render(<Digit digit={12} />)
    const title = screen.getByText(/12/i)
    expect(title).toBeInTheDocument()
    expect(title).toMatchSnapshot()
  })
})
