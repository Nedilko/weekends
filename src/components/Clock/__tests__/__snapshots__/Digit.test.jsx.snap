// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Digit should render digit 1`] = `
<div
  class="flex h-20 items-center justify-center text-5xl font-thin"
>
  12
</div>
`;
